front_end/data.js

```diff
@@ -0,0 +1,10 @@
+const list_questions = {
+    0: "Favorite verse?", 
+    1: "Which Bible character do you relate the most?",
+    2: "How do you obey God while honoring your parents?",
+    3: "Favorite worship song?",
+    4: "Your love language?",
+    5: "Do you love God?"
+};
+
+export { list_questions };
```

Comment:
I don't think you should commit these temporary files. Imo it's better to use an imaginary REST api instead of hardcoded data

Reply:
Also I don't even see where it's being used?

Reply:
oh you're right. I think I removed the code that used the data.